added special case for windows OS recommendation (#104332)
Azure/azure-content: articles/sentinel/sap/configure-audit.md

1. If you changed any settings in the **Security Audit Log Configuration - Parameter** section, select **Save** to save the changes. Auditing will be activated only after the server is rebooted.

> ** Important note** - SAP applications running on Windows OS should consider recommendations in SAP Note 2360334 in case audit log is not read correctly after setup.

Reviewer: ```suggestion
   > [!IMPORTANT]
   > SAP applications running on Windows OS should consider recommendations in SAP Note 2360334 in case the audit log isn't read correctly after setup.
```

>     ![Screenshot showing R S A U CONFIG parameters.](./media/configure-audit/rsau-config-parameter.png)

Reviewer: ```suggestion
   ![Screenshot showing R S A U CONFIG parameters.](./media/configure-audit/rsau-config-parameter.png)
```
